mix.exs

```diff
@@ -4,7 +4,7 @@ defmodule SmartCityTest.MixProject do
   def project do
     [
       app: :smart_city_test,
-      version: "0.5.3",
+      version: "0.5.4",
```

Comment:
This would break backwards compatibility with any tests that expect `TDG.create_dataset` to return a `SmartCity.Dataset` object.

I question why we're doing this in the first place, but it's definitely not a patch version change.

Reply:
In our discovery API card that we're working on we need both SmartCity.Registry.Dataset and SmartCity.Organization. We' using Aliases to keep both without naming conventions, and a helper that wraps registry objects and converts them to their non-registry versions. I wonder if something like that would work here as a temporary stop gap maneuver.